src/SignalRServiceExtension/Config/OptionsSetup.cs

```diff
@@ -39,7 +39,7 @@ public OptionsSetup(IConfiguration configuration, ILoggerFactory loggerFactory,
 
         public void Configure(ServiceManagerOptions options)
         {
-            options.ConnectionString = configuration[connectionStringKey];
+            options.ConnectionString = configuration.GetConnectionString(connectionStringKey) ?? configuration[connectionStringKey];
```

Comment:
Seems no related to the PR. And we don't want to support too many configuration styles, otherwise it is a little messy.